content/en/docs/ibm/pipelines.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Pipelines on IBM Cloud Kubernetes"
+description = "Instructions for using Kubeflow Pipelines on IBM Cloud Kubernetes"
+weight = 50
++++
+
+## Authenticating Pipelines with the SDK
+
+**Notes**:
+* Python package `kfp` v1.0.0 is required.
+* This feature is available with multi-user, auth-enabled Kubeflow installation deployed from the manifest [https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_ibm_dex_multi_user.v1.1.0.yaml](https://raw.githubusercontent.com/kubeflow/manifests/v1.1-branch/kfdef/kfctl_ibm_dex_multi_user.v1.1.0.yaml).
+* Since it transports sensitive info like session cookie value over edge network, it highly recommends enabling HTTPS for the public endpoint of Kubeflow.
```

Comment:
```suggestion
* Since it transports sensitive information like session cookie value over edge network, we highly recommend enabling HTTPS for the public endpoint of Kubeflow.
```